wca-regulations.md

```diff
@@ -141,8 +141,10 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3j2) Definition: Two pieces are similar to each other if they are identical in shape and size, or mirrored in shape and identical in size.
     - 3j3) Corrugated/textured parts which permit the orientation of pieces to be distinguished by feel are not permitted for blindfolded events.
 - 3k) Puzzles should be approved by the WCA Delegate before use in the competition.
-    - 3k1) If a puzzle is found to be not permitted during a round, the competitor must submit a replacement.
-    - 3k2) Penalty for attempts done with puzzles which are not permitted: disqualification of the attempt (DNF). Exception: If a puzzle is found to be not permitted during a round, earlier results may be replaced with an extra attempt, at the discretion of the WCA Delegate.
+    - 3k1) If a puzzle is found to be not permitted during a round, the competitor must submit a replacement. Exception:
+    - 3k2) Penalty for attempts done with puzzles which are not permitted: disqualification of the attempt (DNF), subject to the following exeptions:
+      - 3k2a) If it is found during a round that a competitor accidentally used a puzzle that is not permitted, their affected results may be replaced with extra attempts, at the discretion of the WCA Delegate. 
```

Comment:
Maybe just copy-paste the original line? It doesn't contain the word "accidentally" which is hard to judge.

"If a puzzle is found to be not permitted during a round, earlier results may be replaced with an extra attempt, at the discretion of the WCA Delegate."